x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -40,6 +40,15 @@ export enum AggregationType {
   P99 = '99th',
 }
 
+export const INTERVAL_MULTIPLIER_FOR_LOOKBACK = 5;
+
+export interface PreviewChartResponseItem {
+  name: string;
+  data: Array<{ x: number; y: number | null }>;
+}
+
+export type PreviewChartResponse = PreviewChartResponseItem[];
```

Comment:
`PreviewChartResponseItem` does not need to be exported (only used here) and `PreviewChartResponse` is only used on the server so not needed in common (can go into `server` folder)

Reply:
`PreviewChartResponseItem` is used in tests too. Moved `PreviewChartResponse` to server.